### PR TITLE
chore: create .prettierrc.json

### PR DESCRIPTION
**Pull request by mschwrdtnr**
create a settings-file for prettier

This will not solve issue #350 !

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
Please remove the added settings from the code workspace so they do not duplicate.

**Comment by Zeroks77** [comment]
Do we need to set the "tabWidth" ? would it effect any file  we currently have ? @DanielHabenicht

**Comment by DanielHabenicht** [comment]
@Zeroks77 depends on what you set it to. Is it causing trouble because it is formatted differently, than add it to the settings.

File: phonebook.code-workspace [hunk 1]
```diff
@@ -43,11 +43,6 @@
     "typescript.preferences.quoteStyle": "single",
     "javascript.referencesCodeLens.enabled": true,
     "typescript.referencesCodeLens.enabled": true,
-    "prettier.jsxSingleQuote": true,
-    "prettier.singleQuote": true,
-    "prettier.tslintIntegration": true,
```
Comment (DanielHabenicht) [inline phonebook.code-workspace]: You are sure to remove this?

**Comment by Zeroks77** [comment]
@DanielHabenicht  & @mschwrdtnr  then let's set it befor something crazy happen :D or do we wanna w8 and see if is needed ?

File: phonebook.code-workspace (hunk 1 above)
Comment (mschwrdtnr) [reply phonebook.code-workspace]: _"This setting has been deprecated. See: https://github.com/prettier/prettier-vscode#vscode-eslint-and-tslint-integration"_

Comment (DanielHabenicht) [reply phonebook.code-workspace]: Ok, 👍

**Pull request merged**